basic_flutter/lib/repository/data_provider.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+import '../data/employee.dart';
+
+class DataProvider {
+  late List<Employee> response;
+
+  Future getEmployeesList(String endpoint) async {
+    await http.post(Uri.parse(endpoint),
+        body: [], headers: {"Content-Type": "application/json"}).then((resp) {
```

Comment:
this is an async function you don't need .then here, but you can do it like that, if you want.

Reply:
I still see it a better fit to use this future execution. I'm saving initializing a new variable which I'm already getting from the response of the request.